platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/gossip/sync/SyncManagerImpl.java

```diff
@@ -182,15 +170,7 @@ public List<NodeId> getNeighborsToCall() {
                 continue;
             }
 
```

Comment:
this whole method is only called by a unit test, so lets delete the whole thing

Reply:
Nice. Deleting this method also allowed me to delete a lot of other code as well.